server/services/departureService.ts

```diff
@@ -29,4 +34,13 @@ export default class DepartureService {
 
     return { ...departure, dateTime: parseISO(departure.dateTime).toLocaleDateString('en-GB') }
   }
+
+  async getReferenceData(): Promise<DepartureReferenceData> {
+    const referenceDataClient = this.referenceDataClientFactory(this.token)
+    return {
+      departureReasons: await referenceDataClient.getReferenceData('departure-reasons'),
+      moveOnCategories: await referenceDataClient.getReferenceData('move-on-categories'),
+      destinationProviders: await referenceDataClient.getReferenceData('destination-providers'),
```

Comment:
Guess there's an opportunity to load these in parallel with `Promise.all`

Reply:
Yeah, nice. I'll do that 👍